### PR TITLE
Updating CMake Readme

### PR DESCRIPTION
**Pull request by Carmezim**
Adding 0.12r updates and pip installation instructions #5942.

### CODE REVIEW

**Comment by Carmezim** [comment]
Should I close this PR based [here](https://github.com/tensorflow/tensorflow/issues/5942#issuecomment-265613813)?

File: tensorflow/contrib/cmake/README.md [hunk 1]
```diff
@@ -13,23 +13,59 @@ Linux.
...
 The CMake files in this directory can build the core TensorFlow runtime, an
 example C++ binary, and a PIP package containing the runtime and Python
 bindings.
 
-Note: Windows support is in an **alpha** state, and we welcome your feedback.
-
 ### Pre-requisites
 
-* CMake version 3.5 up to 3.6
+* CMake versions 3.5 and [3.6](https://cmake.org/files/v3.6/cmake-3.6.3-win64-x64.msi)
```
Comment (mrry) [inline tensorflow/contrib/cmake/README.md]: Can you remove the link here, and add a line under "known-good configurations" for Windows below?

File: tensorflow/g3doc/get_started/os_setup.md [hunk 2]
```diff
@@ -468,7 +467,8 @@ export CUDA_HOME=/usr/local/cuda
 
 ### Run TensorFlow from the Command Line
 
-See [common problems](#common-problems) if an error happens.
+See [
+blems](#common-problems) if an error happens.
```
Comment (mrry) [inline tensorflow/g3doc/get_started/os_setup.md]: Typo?

File: tensorflow/g3doc/get_started/os_setup.md [hunk 3]
```diff
@@ -983,6 +983,28 @@ Exception:
...
+
+2. Install TensorFlow again adding `--ignore-installed` flag:  
+`pip install --ignore-installed --upgrade <tensorflow_url>`
+
+
+
+Step #1 might already solve the problem, however if it still persists, execute step #2.
+
+This is an issue that occurs with new Ananconda installations when `pip` tries to remove `easy-install.pth` 
+which does not exist as conda packages cannot include it.
```
Comment (mrry) [inline tensorflow/g3doc/get_started/os_setup.md]: Nit: replace lines 1004--5 with:

This issue occurs with new Anaconda installations when `pip` tries to remove `easy-install.pth`. This file is not included in Anaconda packages, which causes the `pip` installation to fail.

File: tensorflow/contrib/cmake/README.md [hunk 4]
```diff
@@ -13,23 +13,59 @@ Linux.
...
+The supported languages include Python 3.5 64-bit (from Python.org or Anaconda) and C++. 
+This release enables acceleration with CUDA 8.0 and cuDNN 5.1 compatible GPUs.
+A [pip package](https://pypi.python.org/pypi/tensorflow) was published and the
+installation can be done with a single command:
+
+`C:\> pip install tensorflow`
+
+And for GPU support:
+
+`C:\> pip install tensorflow-gpu`
```
Comment (mrry) [inline tensorflow/contrib/cmake/README.md]: I'd remove the content here (lines 16--27) and replace it with something like the following:

CMake can be used to build TensorFlow on Windows. See the [getting started documentation](https://www.tensorflow.org/get_started/os_setup.html#pip-installation-on-windows) for instructions on how to install a pre-built TensorFlow package on Windows.

File: tensorflow/contrib/cmake/README.md [hunk 5]
```diff
@@ -13,23 +13,59 @@ Linux.
...
+ - QuantizedMaxPoo
+ - QuantizeDownAndShrinkRange
+ - QuantizedRelu
+ - QuantizedRelu6
+ - QuantizedReshape
+ - QuantizeV2
+ - RequantizationRange
+ - Requantize
+
+For further details on 0.12r, see the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) and [Google Blog post](https://developers.googleblog.com/2016/11/tensorflow-0-12-adds-support-for-windows.html).
```
Comment (mrry) [inline tensorflow/contrib/cmake/README.md]: Delete this line.

File: tensorflow/g3doc/get_started/os_setup.md [hunk 6]
```diff
@@ -35,8 +35,7 @@ If you are familiar with Pip, Virtualenv, Anaconda, or Docker, please feel free
 to adapt the instructions to your particular needs.  The names of the pip and
 Docker images are listed in the corresponding installation sections.
 
-If you encounter installation errors, see
-[common problems](#common-problems) for some solutions.
+If you encounter installation errors, see [common problems](#common-problems) for some solutions.
```
Comment (mrry) [inline tensorflow/g3doc/get_started/os_setup.md]: Nit: Revert the change here.

**Review by mrry — CHANGES_REQUESTED** [review]
I think some of the edits are useful... especially the common installation problems ones. I've requested a few changes which should make things clearer. Thanks for submitting this!

**Comment by Carmezim** [comment]
@mrry So sorry Derek, I meant the CMake docs one as I made a separate PR earlier for "common problems", #6167, which was merged already. I should've rolled back those last commits, very sorry. But as you suggested those changes I will update accordingly.

**Comment by mrry** [comment]
No problem! If you'd like to rebase on top of your changes and apply the suggestions, that'd be great!

File: tensorflow/contrib/cmake/README.md (hunk 4 above)
Comment (Carmezim) [inline tensorflow/contrib/cmake/README.md]: Absolutely! Much conciser. I ended up placing these notes because it was right after the pre-release and I am not sure if there was an updated official doc already on tensorflow.org. 
Will replace as you've said.

File: tensorflow/contrib/cmake/README.md (hunk 1 above)
Comment (Carmezim) [inline tensorflow/contrib/cmake/README.md]: Sure!

File: tensorflow/contrib/cmake/README.md (hunk 5 above)
Comment (Carmezim) [inline tensorflow/contrib/cmake/README.md]: 29-50 right?

File: tensorflow/g3doc/get_started/os_setup.md (hunk 3 above)
Comment (Carmezim) [inline tensorflow/g3doc/get_started/os_setup.md]: much better.

File: tensorflow/g3doc/get_started/os_setup.md (hunk 2 above)
Comment (Carmezim) [inline tensorflow/g3doc/get_started/os_setup.md]: Yes, that was it.

**Comment by mrry** [comment]
Oops, I think something went wrong with the rebase and now it's pulling in a 60k-line diff! I'm not sure how to back out from this, but maybe it'd be easiest to open a new PR against master?

**Comment by Carmezim** [comment]
@mrry Oh no! 60k lines? That's terrible, I just dropped those two commits and reworded the last. I am really sorry. You can close this PR and I will open a new one. Apologies for the confusion.

**Comment by mrry** [comment]
Don't worry about it! I think the pull request interface is easily confused :). Looking forward to your new PR!

**Comment by Carmezim** [comment]
I feel terrible because I know it is really annoying losing time with this kind of mistake and something simple and you have a lot to worry about and do already. That was embarrassing.  I am really sorry. Thank  you for the patience and being so nice.
